editors/vscode/package.json

```diff
@@ -198,8 +198,9 @@
     "@vscode/test-electron": "^2.4.1",
     "@vscode/vsce": "^3.0.0",
     "cross-env": "^10.0.0",
-    "esbuild": "^0.25.0",
+    "glob": "^11.0.3",
     "ovsx": "^0.10.0",
+    "rolldown": "^1.0.0",
```

Comment:
Please force `1.0.0-beta.35`. 
I get some native binding bugs with newer version, will look into it.

If renovate can update with a newer version, my machine is wrong :)